src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -144,9 +144,15 @@ daos_tests:
     test_Z:
       daos_test: Z
       test_name: DAOS aggregate ec tests
-    test_X:
+    test_X_1:
       daos_test: X
-      test_name: DAOS degraded ec tests
+      test_name: DAOS degraded ec tests 0-6
+      args: -u subtests="0-6"
+      test_timeout: 900
```

Comment:
is full 900 needed?

Reply:
ah, probably not. I will shrink it if it needs to refresh